README.md

```diff
@@ -14,6 +14,26 @@ What's this new konnector?
 The konnector gets information from Malakoff Mederic (french national insurance)
 This konnector fetches the list of reimbursements from [Malakoff Mederic][malakoff_mederic].
 
+What data is imported ?
```

Comment:
I would remove "What's this new konnector" since it is not new anymore and put the two description sentence below.

s/french/French/